lib/freight/apt.sh

```diff
@@ -1,4 +1,8 @@
-TTY="$(tty -s && echo "1" || :)"
+#!/usr/bin/env sh
```

Comment:
I don't know if we want to add this - perhaps just disable the check by putting `shellcheck shell=sh` here instead?